wear/src/main/java/io/homeassistant/companion/android/theme/Color.kt

```diff
@@ -44,3 +46,11 @@ internal val wearColorScheme: ColorScheme = ColorScheme(
     outline = md_theme_dark_outline,
     outlineVariant = md_theme_dark_outlineVariant
 )
+
+@Composable
```

Comment:
I opted to add this here as its a theme color for the switch and might be easier to reference if it was all in the same location? I can move it if we feel thats better.

Reply:
Makes sense as a location for now